lib/coradoc/element/author.rb

```diff
@@ -13,10 +13,10 @@ def initialize(first_name, last_name, email, middle_name = nil)
       end
 
       def to_adoc
-        adoc = "#{@first_name} "
-        adoc << "#{@middle_name} " if @middle_name
-        adoc << @last_name.to_s
-        adoc << " <#{@email}>" if @email
+        adoc = "#{@first_name}"
+        adoc << " #{@middle_name}" if @middle_name
+        adoc << " #{@last_name.to_s}"
+        adoc << " <#{@email}>\n" if @email
```

Comment:
Use multiline string.

Reply:
This would make the code much more unreadable; this code returns a single line.